client/signup/steps/woocommerce-install/confirm/index.tsx

```diff
@@ -18,15 +18,32 @@ import {
 	EligibilityData,
 } from 'calypso/state/automated-transfer/selectors';
 import { getSiteDomain } from 'calypso/state/sites/selectors';
-import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 import type { WooCommerceInstallProps } from '../';
+import type { GoToStep } from '../../../types';
 
 import './style.scss';
 
+function StepPicture( {
```

Comment:
Do we still need this, since we're [removing the images for now](https://github.com/Automattic/wp-calypso/pull/58024#issuecomment-977556114)?

Reply:
Not sure where this came from, can probably rebase it out.

Reply:
should we create an issue for this? or just tackle it here?

Reply:
let's do it here https://github.com/Automattic/wp-calypso/issues/58331

Reply:
> Yes, looks like we should remove this now.

👍

Reply:
Addressed [here](https://github.com/Automattic/wp-calypso/commit/93ca9ebf5b4497ad146060f72b1f9b36e63542dc). Feel free to cherry-pick it 🍒

Reply:
Thanks, done.